src/vos/vos_layout.h

```diff
@@ -75,8 +75,8 @@ POBJ_LAYOUT_BEGIN(vos_pool_layout);
 
 POBJ_LAYOUT_ROOT(vos_pool_layout, struct vos_pool_df);
 POBJ_LAYOUT_TOID(vos_pool_layout, struct vos_cont_df);
-POBJ_LAYOUT_TOID(vos_pool_layout, struct vos_dtx_table_df);
-POBJ_LAYOUT_TOID(vos_pool_layout, struct vos_dtx_entry_df);
+POBJ_LAYOUT_TOID(vos_pool_layout, struct vos_dtx_act_ent_df);
+POBJ_LAYOUT_TOID(vos_pool_layout, struct vos_dtx_cmt_ent_df);
 POBJ_LAYOUT_TOID(vos_pool_layout, struct vos_dtx_record_df);
 POBJ_LAYOUT_TOID(vos_pool_layout, struct vos_obj_df);
 POBJ_LAYOUT_TOID(vos_pool_layout, struct vos_krec_df);
```

Comment:
I think these all can be removed.   They serve no useful purpose if we aren't using typed oids

Reply:
OK, I will remove them.